elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/schema/SQLSchema.java

```diff
@@ -63,24 +63,25 @@ public SQLSchema(Class<?> entityClass, EntityDictionary dictionary) {
     }
 
     @Override
-    protected Dimension constructDimension(String dimensionField, Class<?> cls, EntityDictionary entityDictionary) {
-        Dimension dim = super.constructDimension(dimensionField, cls, entityDictionary);
+    protected DimensionColumn constructDimension(String dimensionField,
+                                                 Class<?> cls, EntityDictionary entityDictionary) {
+        DimensionColumn dim = super.constructDimension(dimensionField, cls, entityDictionary);
 
         JoinTo joinTo = entityDictionary.getAttributeOrRelationAnnotation(cls, JoinTo.class, dimensionField);
 
         if (joinTo == null) {
             String columnName = getColumnName(entityClass, dimensionField);
 
-            if (dim instanceof TimeDimension) {
-                return new SQLTimeDimension(dim, columnName, getAlias());
+            if (dim instanceof TimeDimensionColumn) {
+                return new SQLTimeDimension((TimeDimensionColumn) dim, columnName, getAlias());
```

Comment:
Can we rename `SQLTimeDimension` -> `SQLTimeDimensionColumn` to match?

Reply:
👍